.github/workflows/slack-notifications.yml

```diff
@@ -148,6 +151,7 @@ jobs:
   failure:
     name: Failure notifications
     runs-on: ubuntu-latest
+    permissions: {}
```

Comment:
Will this still allow access to `secrets.SLACK_GHA_*_WEBHOOK` secrets that contain the URLs to submit the request to? I'm not finding the level of access required to read secrets in the documentation.